src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -22,7 +23,7 @@ public ResponseEntity<String> addTelevision(@RequestBody Television television)
     }
 
     @PutMapping("/televisions/{id}")
```

Comment:
Je geeft hier "/television" al eerste deel van het pad, maar dat doe je bij elke mapping. Het is dan handiger om bovenaan je klasse de annotatie `@`RequestMapping("/televisions") toe te voegen. Dan hoef je dat stukje niet bij elke mapping opnieuw te typen. Het zou jammer zijn als je perongeluk bij een van de mappings een typfoutje maakt, of je besluit achteraf toch dat het "/screens" moet zijn